src/nvim/eval/funcs.c

```diff
@@ -8168,6 +8168,17 @@ static void f_setcharsearch(typval_T *argvars, typval_T *rettv, FunPtr fptr)
   }
 }
 
+/// "setcmdline()" function
+static void f_setcmdline(typval_T *argvars, typval_T *rettv, FunPtr fptr)
```

Comment:
let's take cmdpos as an optional second parameter (`setcmdpos()` will not work, it needs to be changed directly in `set_cmdline_str`)

Reply:
Fixed.